app/components/items/download-qr-code.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from "react";
+import { motion } from "framer-motion";
+import { atom, useAtom } from "jotai";
+import { tw } from "~/utils";
+import { CrossIcon } from "../icons";
+import { Button } from "../shared";
+
+const sidebarStatusAtom = atom(false);
+const toggleSidebarAtom = atom(
+  (get) => get(sidebarStatusAtom),
+  (get, set) => set(sidebarStatusAtom, !get(sidebarStatusAtom))
+);
+
+const DownloadQrCode = () => {
+  const [isSidebarOpen, toggleSidebar] = useAtom(toggleSidebarAtom);
+
+  useEffect(()=>{
+    if(isSidebarOpen) {
+      document.body.classList.add("overflow-hidden")
```

Comment:
What is the purpose of this? Is it to disabled the scrolling when the sidebar is open? 
I any case, using document is not okey. Especially considering we will have a URL that corresponds to the sidebar being open, so that means there will be hydration issues. Can you please just comment that line and once i know how the back-end will be build we will solve it. Thanks

Reply:
Yes, the purpose is to disable scrolling when the sidebar is open.
Ahhh yes, we will be having a URL that corresponds to the sidebar being open. so, okay will comment this out for now.
Thanks.